### ISSUE TITLE
Exception when loading relationships in an update

### ISSUE THREAD

**Issue by hanrelan** [ash-project/ash_admin#64]
**Describe the bug**
After updating to the latest `ash_admin`, I can no longer load relationships in an update action.

For the following action:
```elixir
    update :manage_templates do
      accept []
      argument :templates, {:array, :map}

      change manage_relationship(:templates, type: :direct_control)
    end
```
When I click the `Load` button in the admin panel for templates, I get the following exception:
```
** (AshPhoenix.Form.NoFormConfigured) templates at path [] must be configured in the form to be used with `inputs_for`. For example:

Available forms:

Example Setup:

    Form.for_create(
[truncated]
    (ash_phoenix 1.2.16) lib/ash_phoenix/form/form.ex:4429: Phoenix.HTML.FormData.AshPhoenix.Form.to_form/4
    (phoenix_live_view 0.19.5) lib/phoenix_component.ex:2364: Phoenix.Component."inputs_for (overridable 1)"/1
    (phoenix_live_view 0.19.5) lib/phoenix_live_view/tag_engine.ex:68: Phoenix.LiveView.TagEngine.component/3
    (ash_admin 0.9.3) lib/ash_admin/components/resource/form.ex:348: anonymous fn/2 in AshAdmin.Components.Resource.Form.render_relationship_input/5
```

**Expected behavior**
It would allow me to add and remove templates. 
**Runtime**
 - Elixir version: 1.15
 - Erlang version: 25
 - OS: Mac OS
 - Ash version: 
 - any related extension versions: 
 ash_admin: 0.9.3
ash_phoenix: 1.2.16
ash: 2.10.2

**Additional context**

In this particular example, the template array was empty when I hit the error.

**Comment by hanrelan** [comment]
Sorry - my mistake. This was due to the old version of ash. Updating to 2.13.3 fixed it!

**Comment by zachdaniel** [comment]
There was an issue where if you changed the update action it would not modify the form appropriately, and I just pushed a fix for that to main as well.

**Comment by zachdaniel** [comment]
released as 0.9.4
